mlir-tensorrt/executor/lib/CAPI/Runtime/Runtime.cpp

```diff
@@ -146,6 +147,48 @@ static MTRT_Status wrap(const Status &status) {
 // MTRT_Stream
 //===----------------------------------------------------------------------===//
 
+static void mlirEnableGlobalDebug(bool enable) { llvm::DebugFlag = enable; }
```

Comment:
@christopherbate I have copied the implementation from https://github.com/llvm/llvm-project/blob/main/mlir/lib/CAPI/Debug/Debug.cpp#L18. I tested it but for some reason it, it is still not dumping anything.
This is how I am using the API: https://github.com/NVIDIA/TensorRT-Incubator/pull/291/files#diff-765df7b86eaf815fdd1a6331a3d7e7c177214cc01aa96fa8040c87850d403806R77

Reply:
That makes sense.... However there's no point adding these functions `mlirXYZ` here. That's very confusing. Just  inline al these static functions you have defined.